internal/pkg/externalplugins/merge/merge.go

```diff
@@ -22,15 +22,16 @@ const PluginName = "ti-community-merge"
 const githubUpdateCommitter = "web-flow"
 
 var (
-	addCanMergeLabelNotification   = "Can merge label has been added.  <details>Git tree hash: %s</details>"
+	addCanMergeLabelNotification = "This pull request has been accepted and has started to merge. " +
```

Comment:
I think this message reply is incorrect because tide is constantly trying to merge pr, and the can-merge label is just one of the requirements for merging. So I think the original reply is fine.

Reply:
When you're talking about adding or removing `can-merge` label, to the user the bot is actually controlling the merge process. Do you have any reason to let the user know and grasp the underlying mechanism?

Reply:
I think the user must know how label control merge works, because we have other labels that will prevent the pr merge.
You can refer to https://book.prow.tidb.io/components/tide.html.

Reply:
I've improved a little bit. But I still insist on preventing to flood the contributor by all the details of the bot.

Reply:
How about `This PR is marked as mergeable by @committer.`?

Because even if the ci passes, it may be prevented from merging because of other labels.

Reply:
That's a rare case (/merge but with DNM label). If you think it's important, we can comment `Blocked by xxxx label` if other conditions is met.

Reply:
Tide itself will show that prompt in the ci status, so I don't think we need to bother with this reply, and it's not a feature that the merge plugin should be responsible for.

Also not only the DNM label, but also the `hold`, `wip` and `needs-rebase` plugins' labels will prevent pr merging.

Reply:
done

Reply:
Thanks!